process/scToProtocol/stakingToPeer.go

```diff
@@ -349,7 +349,7 @@ func (stp *stakingToPeer) getAllModifiedStates(body block.Body) (map[string]stru
 // PeerChanges returns peer changes created in current round
 func (stp *stakingToPeer) PeerChanges() []block.PeerData {
 	stp.mutPeerChanges.Lock()
-	peersData := make([]block.PeerData, 0)
+	peersData := make([]block.PeerData, 0, len(stp.peerChanges))
 	for _, peerData := range stp.peerChanges {
 		peersData = append(peersData, peerData)
```

Comment:
index instead of append. it can be applied in many places.

Reply:
I prefer to keep it this way, we'll need an additional counter since `stp.peerChanges` is a map